docs/run-prebuilt-wda.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Run Prebuilt WebDriverAgentRunner
+---
+
+
+https://github.com/appium/WebDriverAgent/releases
```

Comment:
I would rather add more detailed description, for example: 
- Open WebDriverAgent releases page
- Locate the build matching your platform
- ...